pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -216,6 +217,17 @@ func BuildStorageFactory(masterConfig configapi.MasterConfig, server *kapiserver
 	// keep Deployments in extensions for backwards compatibility, we'll have to migrate at some point, eventually
 	storageFactory.AddCohabitatingResources(extensions.Resource("deployments"), apps.Resource("deployments"))
 
+	if server.Etcd.EncryptionProviderConfigFilepath != "" {
+		glog.V(4).Infof("Reading encryption configuration from %q", server.Etcd.EncryptionProviderConfigFilepath)
```

Comment:
This log line was added by me. I think it won't hurt but it simplifies a debugging process.